controllers/dspipeline_params.go

```diff
@@ -646,6 +647,25 @@ func (p *DSPAParams) ExtractParams(ctx context.Context, dsp *dspa.DataSciencePip
 			}
 		}
 
+		if p.APIServer.CustomKfpLauncherConfigMap != "" {
+			cm, err := util.GetConfigMap(ctx, p.APIServer.CustomKfpLauncherConfigMap, p.Namespace, client)
```

Comment:
would suggest adding a test to verify this functionality. Otherwise, code changes lgtm.

Reply:
test [here](https://github.com/opendatahub-io/data-science-pipelines-operator/pull/681/files#diff-9d8269106f790367bc28059c0c50f89fc4a07124c33199cb897dcf6923378affR264)